src/mame/drivers/ksys573.cpp

```diff
@@ -43,96 +43,97 @@
+A *GUITARFREAKS 2ndMIX Link ver.                     1999.09
+D GUITARFREAKS 3rdMIX                                2000.04    GE949 JA          949 JA C01 / 949 JA C02
+D GUITARFREAKS 4thMIX                                2000.08    GEA24 JA          A24 JA A02
+D GUITARFREAKS 5thMIX                                2001.03    GCA26 JA          A26 JA A02
+D GUITARFREAKS 6thMIX                                2001.09    GCB06 JA          B06 JA A02
+D GUITARFREAKS 7thMIX                                2002.02    GCB17 JA          B17 JA A02
+N GUITARFREAKS 8thMIX                                2002.08    GCC08 JA          C08 JA A02
+N GUITARFREAKS 8thMIX power-up ver.                  2002.11    GEC08 JB          C08 JB A02
+N GUITARFREAKS 9thMIX                                2003.04    GCC39 JA          C39 JA A02
+N GUITARFREAKS 10thMIX                               2003.10    GCD10 JA          D10 JA A02
+N GUITARFREAKS 11thMIX                               2004.04
+G GunMania                                           2000.07    G?906 JA          (no CD)
+? *GunMania Zone Plus                                2000.10
+P Handle Champ                                       1997.12    GQ710 JA          (no CD)
+P Hyper Bishi Bashi Champ                            1998.07    GC876 EA          (no CD)
+P Hyper Bishi Bashi Champ - 2 Player                 1999.08    GC908 JA          908    A02
+P Jikkyou Powerful Pro Yakyuu EX                     1998.04    GX802 JA          802 JA B02
+P Jikkyou Powerful Pro Yakyuu EX 98                  1998.08
+AA Kick & Kick                                       2001       GNA36 EA          (no CD)
+P Konami 80's Arcade Gallery                         1998.11    GC826 JA          826 JA A01
+P Konami 80's AC Special                             1998       GC826 UA          826 UA A01
+D Mambo a GoGo                                       2001.06
+D Punchmania Hokuto no Ken                           2000.03                      918 JA B02
+D Punchmania Hokuto no Ken 2                         2000.12                      A09 JA A02
 P Salary Man Champ
-P Salary Man Champ - 2 Player                   2001.02    GCA18 JA          A18 JA(needs redump)
-P Step Champ                                    1999.12
+P Salary Man Champ - 2 Player                        2001.02    GCA18 JA          A18 JA(needs redump)
+P Step Champ                                         1999.12
 
 P: plain System573
 A: uses ext. analog I/O board GX700-PWB(F)
+AA: uses alt. ext. analog I/O board GX700-PWB(K)
 D: uses ext. digital sound and I/O board GX894-PWB(B)
-N: uses network PCB unit GUC07 + ext. digital sound and I/O board GX894-PWB(B)
+N : uses network PCB unit GUC07 + ext. digital sound and I/O board GX894-PWB(B)
 G: gun mania only, drives air soft gun (this game uses real BB bullet)
-K: uses alt. ext. analog I/O board GX700-PWB(K)
+K: a modified System 573, equipped with four video input/output and a DAM karaoke device
```

Comment:
Nitpicky about wording but it uses a standard System 573 but with the karaoke I/O board that allows communication with external hardware for the karaoke functionality. 

It is probably to just leave it at "uses karaoke I/O board GX921-PWB(B)(?)" because the I/O board itself is already documented in more detail here: https://github.com/mamedev/mame/blob/master/src/mame/machine/k573kara.cpp